PXWeb/Resources/Styles/main-pxweb.css

```diff
@@ -106,14 +106,36 @@ $scrollbar-color: #b0b0b0;
 /* **********************************  _PxWeb.scss *********************************** */
 h1 {
```

Comment:
The pure element selectors could perhaps go in the _global-styles-on-elements.scss , but perhaps not :-)